news/7796.build.rst

```diff
@@ -1,3 +1,4 @@
-To select a specific compiler for PyInstaller's bootloader, you have two options through environment variables. 
-If the variable ``PYINSTALLER_BOOTLOADER_COMPILER`` exists and has a value of 'gcc' or 'clang', PyInstaller will use the corresponding compiler. 
-Additionally, the presence of the environment variable ``PYINSTALLER_COMPILER_BOOTLOADER`` is necessary to activate this behavior.
\ No newline at end of file
+To enable the passing of extra arguments to the bootloader compiler during installation via pip, 
+you can utilize the environment variable ``PYINSTALLER_BOOTLOADER_WAF_ARGS``. However, 
+it is essential to ensure that the environment variable ``PYINSTALLER_BOOTLOADER_COMPILER`` 

```

Comment:
I'm guessing you really mean
```suggestion
it is essential to ensure that the environment variable ``PYINSTALLER_COMPILE_BOOTLOADER``
```

Reply:
Although I suppose we could remove that caveat and make `PYINSTALLER_BOOTLOADER_WAF_ARGS` imply `PYINSTALLER_COMPILE_BOOTLOADER`?

Reply:
So we should remove the `PYINSTALLER_COMPILE_BOOTLOADER`? Is this what you mean?

Reply:
We shouldn't be removing `PYINSTALLER_COMPILE_BOOTLOADER`; both for backwards compatibility and to be able to rebuild bootloader without any additional options. 

We *could* make it so that non-empty `PYINSTALLER_BOOTLOADER_WAF_ARGS` also activates rebuild of bootloader (even if `PYINSTALLER_COMPILE_BOOTLOADER` is explicitly especified) - but I'm not sure it is worth the hassle.